### PR TITLE
Switched git-rev-sync to use mikkoh/git-rev-sync

### PR DESCRIPTION
**Pull request by mikkoh**
There is currently a bug which exist in https://www.npmjs.com/package/git-rev-sync

I've created a pull request which fixes this issue:
https://github.com/kurttheviking/git-rev-sync/pull/19

Temporarily though I'd like to use my fork as a dep on `get-project-version`.

Thoughts @minasmart @lreeves ?

### CODE REVIEW

**Comment by lreeves** [comment]
Yep makes sense to me.

**Comment by minasmart** [comment]
Sounds good. In the PR you made though: I'd add a comment around what the regex is searching for? Right now, it's just a big 'ol regex that's sort of hard to parse.

**Comment by lreeves** [comment]
Is the repo you're pointing at private? The build fails trying to find that commit.

**Comment by mikkoh** [comment]
No it's not private. Right now the build is failing because of eslint having a hard time locating where plugins exist. 😢 

I've looked into this issue before and haven't been able to figure it out. @minasmart might remember this.

**Comment by mikkoh** [comment]
@minasmart `eslint` not finding the correct deps was fixed by using a newer version of npm on Circle 😬 

I'm sure you'll come across it at some point.

**Comment by minasmart** [comment]
Quite likely, but probably quite some time ago. Good catch!
